tasks/gitlab_permissions.yml

```diff
@@ -24,24 +29,15 @@
     mode: 0750
     state: directory
 
-- name: Link log directory
-  file:
-    src: /var/log/gitlab
-    dest: "{{ gitlab_install_path | mandatory }}/log"
-    state: link
-    owner: git
-
-- name: Create tmp directory
+- name: Delete old log directory
   file:
-    path: "/tmp/gitlab"
-    owner: git
-    mode: 0750
-    state: directory
+    path: "{{ gitlab_install_path | mandatory }}/log"
+    state: absent
```

Comment:
There is no when condion. Won't this delete the link you create in the next task when the role is executed again?
Is deletion even needed? Won't the link override the existing folder, maybe using a force option?

Reply:
no it wont because the folder contains a file called .gitkeep

Reply:
and fixed

Reply:
Does ansible respect `.gitkeep` when deleting files?

Reply:
yes it does .gitkeep is not a standard its pretty much just a file, you only execute this once so there is no when statment needed